packages/roosterjs-content-model-markdown/lib/markdownToModel/creators/createBlockQuoteFromMarkdown.ts

```diff
@@ -0,0 +1,30 @@
+import { createFormatContainer } from 'roosterjs-content-model-dom';
+import { createParagraphFromMarkdown } from './createParagraphFromMarkdown';
+import type {
+    ContentModelFormatContainer,
+    ContentModelFormatContainerFormat,
+} from 'roosterjs-content-model-types';
+
+const QuoteFormat: ContentModelFormatContainerFormat = {
+    borderLeft: '3px solid rgb(200, 200, 200)',
+    textColor: 'rgb(102, 102, 102)',
+    marginTop: '1em',
+    marginBottom: '1em',
+    marginLeft: '40px',
+    marginRight: '40px',
+    paddingLeft: '10px',
+};
+
+/**
+ * @internal
+ */
+export function createBlockQuoteFromMarkdown(
+    text: string,
+    blockquote?: ContentModelFormatContainer
+): ContentModelFormatContainer {
+    text = text.replace('>', '');
```

Comment:
I think we should only replace the first `>`

Reply:
replace function only replace the first element found, to replace all we need to use />/g